tests/schemas/api_v3/networkStatistics.schema.js

```diff
@@ -15,25 +15,27 @@
  */
 import Joi from 'joi';
 
-const goodRequestSchema = {
-	data: Joi.object().required(),
-	meta: Joi.object().required(),
-	links: Joi.object().optional(),
-};
+const regex = require('./regex');
 
 const basicStatsSchema = {
 	totalPeers: Joi.number().integer().min(0).required(),
 	connectedPeers: Joi.number().integer().min(0).required(),
 	disconnectedPeers: Joi.number().integer().min(0).required(),
 };
 
+const totalItem = {
+	amount: Joi.string().pattern(regex.POSITIVE_DIGITS).required(),
+	tokenID: Joi.string().pattern(regex.TOKEN_ID).required(),
+};
```

Comment:
```suggestion
const totalItem = {
	tokenID: Joi.string().pattern(regex.TOKEN_ID).required(),
	amount: Joi.string().pattern(regex.POSITIVE_DIGITS).required(),
};
```

Reply:
I do not see any change in logic. Am I missing something? Otherwise, would prefer keeping it as is.

Reply:
Aesthetics, no change in business logic. Everywhere across the implementation, we state the `tokenID` first followed by the `amount`.

Reply:
Applied 👍